cmd/cmdtest/testutils.go

```diff
@@ -68,6 +67,9 @@ func TestSetup(t *testing.T, parallel bool) {
 }
 
 func TestSetupWithSandbox(t *testing.T, parallel bool) (*TestSandbox, func()) {
+
+	var TestDirPath, _ = filepath.Abs(filepath.Join("..", "cmd", "testdata"))
```

Comment:
Best practice to make local variables start with lower case

Reply:
Yes of course! It's from where I copied and pasted from global scope - rookie mistake, sorry!